testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/conftest.py

```diff
@@ -33,6 +33,8 @@ def params_from_base_suite_setup(request):
     sg_lb = request.config.getoption("--sg-lb")
     use_sequoia = request.config.getoption("--sequoia")
     no_conflicts_enabled = request.config.getoption("--no-conflicts")
+    sg_installer_type = request.config.getoption("--sg-installer-type")
+    sa_installer_type = request.config.getoption("--sa-installer-type")
```

Comment:
I hope system test and upgrade test does not required this params

Reply:
We are not running those tests on windows platforms currently